config.py

```diff
@@ -0,0 +1,19 @@
+config = {
+        "debug":False,
+        "uri":"bolt://localhost:7687",
+        "user": "neo4j",
+        "password":"QM6yBEkttU7VWx",
+        "dry_run":True,
+        "only_norway":True,
+        "only_pages":True,
+        "only_include_desktop":True
+        }
+
+pagelinks_config = {
+    "debug":False,
+    "host":"localhost",
+    "user":"root",
+    "password":"my-secret-pw",
+    "database":"pagelinks-db",
+    "dry_run":True
+}
```

Comment:
```suggestion
import os

config = {
        "debug":False,
        "uri":"bolt://localhost:7687",
        "user": os.environ.get("NEO_USERNAME"),
        "password": os.environ.get("NEO_PASSWORD"),
        "dry_run":True,
        "only_norway":True,
        "only_pages":True,
        "only_include_desktop":True
        }

pagelinks_config = {
    "debug":False,
    "host":"localhost",
    "user": os.environ.get("MARIADB_USER"),
    "password": os.environ.get("MARIADB_ROOT_PASSWORD"),
    "database":"pagelinks-db",
    "dry_run":True
}
```
Kan smelle det inn i .env lokalt og på serveren maybe?